### ISSUE TITLE
Could not load file or assembly System.Configuration.ConfigurationManager even when in BeautyExcludes

### ISSUE THREAD

**Issue by mightyPolo108** [nulastudio/NetBeauty2#48]
Hi.
Iam having issues with assembly System.Configuration.ConfigurationManager on net6/Windows10.

I tried to use BeautyExcludes System.*;Microsoft.* but to no avail.

The error message is 
`Could not load file or assembly 'System.Configuration.ConfigurationManager, Version=7.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'. Could not find or load a specific file. (0x80131621)`

I can see the file is present in output folder, but somehow its not seen by the nbloader?

 I tried sharedmode / excludes nothing worked.

Is there something special about this assembly?
How can I work around this?

My Config
`<BeautySharedRuntimeMode>False</BeautySharedRuntimeMode>
		
		<BeautyLibsDir Condition="$(BeautySharedRuntimeMode) == 'True'">../libraries</BeautyLibsDir>
		<BeautyLibsDir Condition="$(BeautySharedRuntimeMode) != 'True'">./libraries</BeautyLibsDir>
		
		 <BeautyExcludes>System.*.dll;Microsoft.*.dll</BeautyExcludes> 
		
		
		
		<DisableBeauty>False</DisableBeauty>
		
		<BeautyOnPublishOnly>False</BeautyOnPublishOnly>
		
		<BeautyEnableDebugging>False</BeautyEnableDebugging>
		
		
		
		<BeautyUsePatch>False</BeautyUsePatch>
		
		
		<BeautyLogLevel>Info</BeautyLogLevel>
		
		
		`

Thanks.

**Comment by liesauer** [comment]
your config is fine, this should be a bug. can you share a testcase?

**Comment by mightyPolo108** [comment]
I am building a small project in which I hope will be able to demonstrate the problem.
Apparently its connected with Winforms Settings.Designer files.
I will post more, later.

**Comment by liesauer** [comment]
actually the `System.Configuration.ConfigurationManager` location is fine no matter excludes or not. it looks like is the dependencies problem. if use `AssemblyLoadContext.LoadFromAssemblyPath` to load the assembly, it just failed.

still finding out what makes `System.Configuration.ConfigurationManager` behaves specially.

**Comment by liesauer** [comment]
https://github.com/dotnet/runtime/issues/12430#issuecomment-479940281

**Comment by liesauer** [comment]
https://www.cnblogs.com/zhurongbo/p/10423231.html

**Comment by mightyPolo108** [comment]
Thank you.
I failed to create a simple program that would demonstrate the problem.
You are right, that the problem is with ConfigurationManager and packages, that uses it as dependency.
For now, I was able to circumvent the problem by striping my project of these packages
```
System.Configuration.ConfigurationManager
System.DirectoryServices.AccountManagement
Microsoft.Extensions.DependencyInjection
Microsoft.Windows.Compatibility
``` 

I will test behaviour on project with heavily relies on DI and let you know.
Thanks.
